lib/src/main/kotlin/com/swmansion/starknet/service/RequestProcessingService.kt

```diff
@@ -0,0 +1,52 @@
+package com.swmansion.starknet.service
+
+import kotlinx.serialization.json.JsonObject
+import java.util.concurrent.CompletableFuture
+
+internal interface RequestProcessingService {
+    data class Payload(
```

Comment:
This is a Http Payload, it doesn't make sense for other protocols.

Reply:
What if we renamed this interface to indicated it's used for http communication? Since I don't think it's useful at this point to create intermediate implementation for Request payloads.